org.gpodder.gpodder-adaptive.json

```diff
@@ -0,0 +1,52 @@
+{
+  "app-id": "org.gpodder.gpodder-adaptive",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "42",
+  "sdk": "org.gnome.Sdk",
+  "command": "gpodder",
+  "rename-icon": "gpodder-adaptive",
+  "copy-icon": true,
+  "rename-desktop-file": "gpodder.desktop",
+  "finish-args": [
+    "--device=dri",
+    "--socket=x11",
+    "--socket=wayland",
+    "--share=ipc",
+    "--share=network",
+    "--own-name=org.gpodder.*",
+    "--talk-name=org.gtk.vfs.*",
+    "--filesystem=home"
```

Comment:
a select number of xdg- locations should be enough.

Reply:
gPodder is a program which deals with media files downloaded from feeds. The default location for the downloaded files is  `~/gPodder/Downloads`, which is also used by the org.gpodder.gpodder package (it has the `--filesystem=home` arg). Having a different location for downloaded files would be really inconvenient for people who want test this version with their existing installation.

Reply:
then explictely allow that folder.